ex03.exs

```diff
@@ -51,11 +51,60 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    # your code goes here
-    # I'm hoping to see a simple pipeline as the body of this function...
+    apply_function(collection, process_count, function)
   end
 
-  # and here...
+  def apply_function(list, n, func) do
+    lists = Enum.map(list, func)
+  end
```

Comment:
Although you added this function to make three of the tests pass, they aren't really passing in the spirit of the exercise, as you're just using plain map.

As pert the rubric, I'm going to have to take 25 off.

# -25